dali/base/dasds.cpp

```diff
@@ -3744,7 +3754,14 @@ int CSDSTransactionServer::run()
             mb.clear();
             if (coven.recv(mb, RANK_ALL, MPTAG_DALI_SDS_REQUEST, NULL))
             {
-                msgCount++;
+                // 1) Need to increment sds_requests here
+                // NB: it will never decremented. This is total for life of this instance.
```

Comment:
trivial: never *be* decremented.  Same in danqs.cpp

Reply:
fixed